addons/binding/org.openhab.binding.bigassfan/src/main/java/org/openhab/binding/bigassfan/internal/handler/BigAssFanHandler.java

```diff
@@ -57,6 +55,32 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static org.openhab.binding.bigassfan.internal.BigAssFanBindingConstants.BAF_PORT;
+import static org.openhab.binding.bigassfan.internal.BigAssFanBindingConstants.CHANNEL_FAN_AUTO;
+import static org.openhab.binding.bigassfan.internal.BigAssFanBindingConstants.CHANNEL_FAN_DIRECTION;
+import static org.openhab.binding.bigassfan.internal.BigAssFanBindingConstants.CHANNEL_FAN_LEARN_MAXSPEED;
+import static org.openhab.binding.bigassfan.internal.BigAssFanBindingConstants.CHANNEL_FAN_LEARN_MINSPEED;
+import static org.openhab.binding.bigassfan.internal.BigAssFanBindingConstants.CHANNEL_FAN_POWER;
+import static org.openhab.binding.bigassfan.internal.BigAssFanBindingConstants.CHANNEL_FAN_SLEEP;
+import static org.openhab.binding.bigassfan.internal.BigAssFanBindingConstants.CHANNEL_FAN_SMARTMODE;
+import static org.openhab.binding.bigassfan.internal.BigAssFanBindingConstants.CHANNEL_FAN_SPEED;
+import static org.openhab.binding.bigassfan.internal.BigAssFanBindingConstants.CHANNEL_FAN_SPEED_MAX;
+import static org.openhab.binding.bigassfan.internal.BigAssFanBindingConstants.CHANNEL_FAN_SPEED_MIN;
+import static org.openhab.binding.bigassfan.internal.BigAssFanBindingConstants.CHANNEL_FAN_WHOOSH;
+import static org.openhab.binding.bigassfan.internal.BigAssFanBindingConstants.CHANNEL_FAN_WINTERMODE;
+import static org.openhab.binding.bigassfan.internal.BigAssFanBindingConstants.CHANNEL_LIGHT_AUTO;
+import static org.openhab.binding.bigassfan.internal.BigAssFanBindingConstants.CHANNEL_LIGHT_COLOR;
+import static org.openhab.binding.bigassfan.internal.BigAssFanBindingConstants.CHANNEL_LIGHT_HUE;
+import static org.openhab.binding.bigassfan.internal.BigAssFanBindingConstants.CHANNEL_LIGHT_LEVEL;
+import static org.openhab.binding.bigassfan.internal.BigAssFanBindingConstants.CHANNEL_LIGHT_LEVEL_MAX;
+import static org.openhab.binding.bigassfan.internal.BigAssFanBindingConstants.CHANNEL_LIGHT_LEVEL_MIN;
+import static org.openhab.binding.bigassfan.internal.BigAssFanBindingConstants.CHANNEL_LIGHT_POWER;
+import static org.openhab.binding.bigassfan.internal.BigAssFanBindingConstants.CHANNEL_LIGHT_PRESENT;
+import static org.openhab.binding.bigassfan.internal.BigAssFanBindingConstants.CHANNEL_LIGHT_SMARTER;
+import static org.openhab.binding.bigassfan.internal.BigAssFanBindingConstants.CHANNEL_MOTION;
+import static org.openhab.binding.bigassfan.internal.BigAssFanBindingConstants.CHANNEL_TIME;
+import static org.openhab.binding.bigassfan.internal.BigAssFanBindingConstants.CHARSET;
```

Comment:
Was there a reason why you deleted the wildcard static import line, and replaced it with all these individual imports?

Reply:
IntelliJ did it and I was too lazy to undo it. I guess it's my configured coding style for work. 

In any case, seems like the right thing to do.

Reply:
Please use the wildcard import instead. That's how the ESH formatter is configured and it will be reverted anyhow the next time someone in Eclipse edits and saves this file. 

Maybe you can setup your IntelliJ so it uses the ESH formatter for openHAB/ESH projects instead? See also https://github.com/openhab/openhab2-addons/pull/4530#issuecomment-453646692.

Reply:
Reverted. I'll see if I can set up IntelliJ next time.